### ISSUE TITLE
[Bug]: CANNOT LINK EXECUTABLE "rustc": cannot locate symbol "LLVMConstExtractValue" 

### ISSUE THREAD

**Issue by ar-cetitec** [termux/termux-packages#12039]
### Problem description

After merging [12013](https://github.com/termux/termux-packages/pull/12013), `rustc` fails with an error from the dynamic linker.

Related: #12008

### What steps will reproduce the bug?

$ rustc --version
CANNOT LINK EXECUTABLE "rustc": cannot locate symbol "LLVMConstExtractValue" referenced by "/data/data/com.termux/files/usr/lib/librustc_driver-1a7b7f67c04da79d.so"...

### What is the expected behavior?

$ rustc --version
rustc 1.63.0

### System information

termux-info:

```
QR-Code:Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=18169
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.dev/apt/termux-main stable main
Updatable packages:
All packages up to date
termux-tools version:
1.29.2
Android version:
12
Kernel build information:
Linux localhost 5.4.147-qgki-gc13e393e64be #1 SMP PREEMPT Sat Jun 25 12:52:59 CST 2022 aarch64 Android
Device manufacturer:
OnePlus
Device model:
LE2123

```

**Comment by xtkoba** [comment]
Please don't expect Rust to work correctly on Termux until a version is released that officially supports LLVM 15.

**Comment by ar-cetitec** [comment]
Seems to be fixed with #12040: did a quick check with `cargo build` of [rage](https://github.com/str4d/rage).

**Comment by xtkoba** [comment]
:+1:

Now that <a href="https://blog.rust-lang.org/2022/09/22/Rust-1.64.0.html">Rust 1.64.0</a> has been releaed, we will not handle problems for 1.63.0 anymore, unless it is extremely hard to bump to 1.64.0. Thanks.
